sw-precache-config.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+  staticFileGlobs: [
+    'out/*.html',
+    'out/static/*',
+    'out/_next/**/*'
+  ],
+  stripPrefix: 'out/',
```

Comment:
I think the cached path is wrong. It seems that the service worker tries to cache files from `/` rather than from `/2.0.0/`.

![image](https://cloud.githubusercontent.com/assets/193136/26245797/0ef77878-3cc0-11e7-9b90-165a6d7a3f75.png)

To fix, add `'replacePrefix': '/2.0.0/'`